projects/plugins/jetpack/extensions/shared/block-category.native.js

```diff
@@ -0,0 +1,28 @@
+import { getCategories, setCategories, registerBlockCollection } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import { JetpackLogo } from './icons';
+
+registerBlockCollection( 'jetpack', {
+	title: 'Jetpack powered',
```

Comment:
Should this be translated, like the other titles?

```suggestion
	title: __( 'Jetpack powered', 'jetpack' ),
```

Reply:
Yes. Thank you for catching that!